modules/initrd.nix

```diff
@@ -161,7 +161,10 @@ let
   extraUtils = mkExtraUtils {
     name = "${device_name}-extra-utils";
     packages = [
-      busybox
+      (busybox.override {
+        enableStatic = true;
+        useMusl = true;
+      })
```

Comment:
This shouldn't be needed. This is odd.

But, I seem to recall dealing with something similar recently but I don't remember what it was.